### ISSUE TITLE
gray theme for hwmonitor

### ISSUE THREAD

**Issue by cgvirus** [linuxmint/cinnamon-spices-applets#2620]
hwmonitor@sylfurd
This excellent applet should have a gray theme for modern interface.
Here is an example:

[hwmonitor_gray theme.json.zip](https://github.com/linuxmint/cinnamon-spices-applets/files/3694907/hwmonitor_gray.theme.json.zip)

**Comment by Hultan** [comment]
Thanks @cgvirus, I will look into that as soon as I can...personally I suck at everything that have to do with how things look (colors and stuff), so this is awesome :-)

**Comment by cgvirus** [comment]
Here are my Final tweaks. With images:
![Gey BlueTheme](https://user-images.githubusercontent.com/22106885/66342797-11730080-e96c-11e9-9157-c66b3966a7c8.jpg)
![GeyTheme](https://user-images.githubusercontent.com/22106885/66342798-11730080-e96c-11e9-9427-fccf43e8cc2a.jpg)

[Hw Monitor Grey Theme.zip](https://github.com/linuxmint/cinnamon-spices-applets/files/3699132/Hw.Monitor.Grey.Theme.zip)

**Comment by Hultan** [comment]
@cgvirus I have implemented your themes now, I hope I got the colors right. I had to do some math to convert them into code, but take a look at them and I will fix any mistakes I have made later.

I have cleaned up the configuration dialog, so I have divided the themes into two sections, general theme (border colors etc) and graph data theme, so you will have to change both to get your themes. Under general theme you will find a "gray theme" and under graph data theme you will find a "blue theme" and a "gray theme".
